python/taichi/lang/ast/checkers.py

```diff
@@ -86,14 +86,11 @@ def generic_visit(self, node):
     def visit_For(self, node):
         # TODO: since autodiff is enhanced, AST checker rules should be relaxed. This part should be updated.
         return
-        if (isinstance(node.iter, ast.Call)
-                and isinstance(node.iter.func, ast.Attribute)
-                and isinstance(node.iter.func.value, ast.Name)
-                and node.iter.func.value.id == 'ti'
-                and node.iter.func.attr == 'static'):
-            is_static = True
-        else:
-            is_static = False
+        is_static = bool(isinstance(node.iter, ast.Call)
```

Comment:
why add bool() here?
maybe use all?
```python
all([a, b, c])
```

Reply:
I use bool according to the log of pylint.
`R1703: The if statement can be replaced with 'var = bool(test)'`
Do we need change the code into the `all([a, b, c])` format?

Reply:
@ailzhang  Thanks. I will fix that.